include/CL/sycl/detail/global_config.hpp

```diff
@@ -54,22 +54,34 @@
 // But the triSYCL OpenCL implementation is actually based on Boost.Compute
 #include <boost/compute.hpp>
 /// A macro to keep some stuff in OpenCL mode
-#define TRISYCL_SKIP_OPENCL(x) x
+// Use variadic macros so we can have straw commas in the argument
```

Comment:
stray?

Reply:
Ah yes. Thank you for improving my English :-)